discretize/base/base_mesh.py

```diff
@@ -466,324 +558,273 @@ def rotation_matrix(self):
         between local and global coordinate systems and provide a tool for
         switching between the two while still maintaing the connectivity of the
         mesh's cells. For a visual example of this, please see the figure in the
-        docs for the :class:`~discretize.mixins.vtkModule.InterfaceVTK`.
+        docs for the :class:`~discretize.mixins.vtk_mod.InterfaceVTK`.
         """
         return np.array([self.axis_u, self.axis_v, self.axis_w])
 
-
     reference_system = properties.String(
-        'The type of coordinate reference frame. Can take on the values ' +
-        'cartesian, cylindrical, or spherical. Abbreviations of these are allowed.',
-        default='cartesian',
-        change_case='lower',
+        "The type of coordinate reference frame. Can take on the values "
+        + "cartesian, cylindrical, or spherical. Abbreviations of these are allowed.",
+        default="cartesian",
+        change_case="lower",
     )
 
     @properties.validator
     def _validate_reference_system(self):
         """Check if the reference system is of a known type."""
-        choices = ['cartesian', 'cylindrical', 'spherical']
+        choices = ["cartesian", "cylindrical", "spherical"]
         # Here are a few abbreviations that users can harnes
         abrevs = {
-            'car': choices[0],
-            'cart': choices[0],
-            'cy': choices[1],
-            'cyl': choices[1],
-            'sph': choices[2],
+            "car": choices[0],
+            "cart": choices[0],
+            "cy": choices[1],
+            "cyl": choices[1],
+            "sph": choices[2],
         }
         # Get the name and fix it if it is abbreviated
         self.reference_system = abrevs.get(self.reference_system, self.reference_system)
         if self.reference_system not in choices:
-            raise ValueError('Coordinate system ({}) unknown.'.format(self.reference_system))
+            raise ValueError(
+                "Coordinate system ({}) unknown.".format(self.reference_system)
+            )
         return True
 
+    # DEPRECATED
+    normals = deprecate_property("face_normals", "normals", removal_version="1.0.0")
+    tangents = deprecate_property("edge_tangents", "tangents", removal_version="1.0.0")
+    projectEdgeVector = deprecate_method(
+        "project_edge_vector", "projectEdgeVector", removal_version="1.0.0"
+    )
+    projectFaceVector = deprecate_method(
+        "project_face_vector", "projectFaceVector", removal_version="1.0.0"
+    )
+
 
 class BaseRectangularMesh(BaseMesh):
     """
     BaseRectangularMesh
     """
 
+    _aliases = {
+        **BaseMesh._aliases,
+        **{
+            "vnC": "shape_cells",
+            "vnN": "shape_nodes",
+            "vnEx": "shape_edges_x",
+            "vnEy": "shape_edges_y",
+            "vnEz": "shape_edges_z",
+            "vnFx": "shape_faces_x",
+            "vnFy": "shape_faces_y",
+            "vnFz": "shape_faces_z",
```

Comment:
ah, I see now that @prisae and my comments above would be addressed here. Do you think we should interchange which is the property and which is the alias?